components/icu/README.md

```diff
@@ -53,11 +53,7 @@ use icu::datetime::{DateTimeFormat, options::length, mock::parse_gregorian_from_
 
 let provider = icu_testdata::get_provider();
 
-let options = length::Bag {
```

Comment:
Question: Is this a required change or just a drive-by style change. If it's required, could you please explain.

Reply:
non_exhaustive doesn't work with FRU (`Struct { field: field, ..foo() }`) syntax

Reply:
It seems to here, unless I'm missing something.

https://play.rust-lang.org/?version=stable&mode=debug&edition=2021&gist=fbf3e21153e6f05569fd4538b6b963cc

Reply:
That's in the same crate, non_exhaustive only applies for cross-crate stuff